src/main/java/io/github/linoxgh/moretools/Items.java

```diff
@@ -1,32 +1,40 @@
 package io.github.linoxgh.moretools;
 
-import java.util.List;
-
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.config.Config;
 import org.bukkit.ChatColor;
 import org.bukkit.Material;
 import org.bukkit.inventory.meta.ItemMeta;
 
-import me.mrCookieSlime.Slimefun.cscorelib2.config.Config;
-import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import java.util.List;
 
 public class Items {
-    
+
     // TOOLS
-    public static final SlimefunItemStack CRESCENT_HAMMER = new SlimefunItemStack("CRESCENT_HAMMER", Material.IRON_HOE, "&bCrescent Hammer", "&7&oActually this is a wrench, really.", "", "&eLeft Click &7> Dismantles the machine.");
+    public static final SlimefunItemStack CRESCENT_HAMMER = new SlimefunItemStack(
+        "CRESCENT_HAMMER",
+        Material.IRON_HOE,
+        "&bCrescent Hammer",
+        "&7&oActually this is a wrench, really.",
+        "",
+        "&eLeft Click &7> Dismantles the machine.");
+
     static {
         Config cfg = MoreTools.getInstance().getCfg();
         ItemMeta meta = CRESCENT_HAMMER.getItemMeta();
         List<String> lore = meta.getLore();
-        
+
         if (cfg.getBoolean("item-settings.crescent-hammer.features.enable-rotation")) {
             lore.add(3, ChatColor.YELLOW + "Right Click " + ChatColor.GRAY + "> Rotates the block, if it's rotatable.");
         }
-        
+
         if (cfg.getBoolean("item-settings.crescent-hammer.features.enable-channel-change")) {
-            lore.add(4, ChatColor.YELLOW + "Shift + Left Click " + ChatColor.GRAY + "> Increases the channel of a cargo node.");
-            lore.add(5, ChatColor.YELLOW + "Shift + Right Click " + ChatColor.GRAY + "> Decreases the channel of a cargo node.");
+            lore.add(4, ChatColor.YELLOW + "Shift + Left Click " + ChatColor.GRAY +
```

Comment:
May I ask why did you divide these? They're not that long and fit your screen. Not to mention that even if you'll divide them at least put the ```ChatColor.GRAY``` in the lower line too.